.travis.yml

```diff
@@ -2,13 +2,13 @@ language:
   php
 
 php:
-  - 5.3
-  - 5.4
   - 5.5
+  - 5.6
+  - 7.0
 
 before_script:
   - curl -s http://getcomposer.org/installer | php
-  - php composer.phar update -v
+  - php composer.phar install -v
```

Comment:
haven't use composer for a very long time, update won't install + update any more?